fcrepo-auth-common/src/main/java/org/fcrepo/auth/common/package-info.java

```diff
@@ -0,0 +1,23 @@
+/**
```

Comment:
License header format should be `/*` instead of `/**`
https://travis-ci.org/fcrepo4/fcrepo4/builds/103756794#L5428